app/javascript/react/components/App.js

```diff
@@ -1,6 +1,7 @@
 import React from 'react'
 import BrandsIndexContainer from "./BrandsIndexContainer"
 import BrandShowContainer from "./BrandShowContainer"
+import PizzaShowContainer from "./PizzaShowContainer"
 import { BrowserRouter, Route, Switch } from "react-router-dom"
```

Comment:
I would recommend starting to separate external from internal imports - grouping the React imports, leave an empty line, then your internal component imports